src/Client/Landingi/ChangeLogClient.php

```diff
@@ -0,0 +1,19 @@
+<?php
+declare(strict_types=1);
+
+namespace Landingi\Core\Client\Landingi;
+
+use JsonSerializable;
+
+interface ChangeLogClient
+{
+    public function getAll(): array;
```

Comment:
Tutaj zawsze mam dylemat. `array`, czy dedykowana kolekcja np: `ChangeLogCollection`?

Reply:
Tylko potem bym musiał zrobić interfejs ChangeLogCollection i jeszcze VO ChangeLog, a nie wiem jak one będą użyte. Jakis pomysł?

Reply:
Bardziej to pytanie na zasadzie "Czy stosować tablice, czy może jednak kolekcje obiektów?"

Reply:
Bardziej chodziło mi o pytanie "Czy stosować tablice, czy jednak kolekcje obiektów?". Takie luźne przemyślenie

Reply:
Może to jest temat na dłuższą rozmowę